docs/getting_started.rst

```diff
@@ -274,8 +274,8 @@ pymemcache and fetch all the needed requirements at run. See the example below:
 You can instantiate all the classes and clients offered by pymemcache.
 
 Your client will remain open until you decide to close it or until you decide
-to quit your interpreter. It can allow you to see what's happen if your server
-is abruptly closed. Below is an by example.
+to quit your interpreter. It can allow you to see what happens if your server
+is abruptly closed. Below is a by-example.
```

Comment:
Out of curiosity can you point me to the grammar guideline for the dash in 'by-example' I haven't personally seen that one before.

Reply:
yeah, sorry! There shouldn't have been '-' but 'by example' instead or 'an example' as suggested by @jparise below.